cpp/include/cudf/fixed_point/fixed_point.hpp

```diff
@@ -24,6 +24,8 @@
 #include <cuda/std/type_traits>
 #include <cuda/std/utility>
 
+#include <string.h>
```

Comment:
This was marked as resolved but the change was not applied. I'll unresolve this. It should also be between `<cmath>` and `<string>` below.

Reply:
Sorry, I've changed this locally but haven't committed the change yet; all of the code in this PR is moving to a new header file (since we wanted it moved out of fixed_point.hpp) but I wanted to wait to commit that until most/all of the issues were resolved.  And since there's so many on the page it's hard to keep track of which ones I haven't done yet, so I've been resolving them just so I could see what was left.

Reply:
My preferred process is to resolve comments I create when I'm satisfied the concern is resolved. If there is continued discussion I thumbs up rather than resolving.

Reply:
I've finally pushed up changes that I think incorporate all of the feedback.  Sorry about mishandling it.